README.md

```diff
@@ -107,11 +107,9 @@ Install python pip modules for dpkt and ipaddr.
 sudo pip install dpkt ipaddr
 ```
 
-Download [python-libpcap](http://sourceforge.net/projects/pylibpcap/files/pylibpcap/0.6.4).
+Install [pypcap](https://github.com/pynetwork/pypcap).
 ```
-tar xvfz pylibpcap-0.6.4.tar.gz
-cd pylibpcap-0.6.4
-sudo python ./setup.py install
+sudo pip install pypcap
```

Comment:
Worth a note about older installs needing to uninstall `pylibpcap`? I'm not sure how often we'll upgrade vs start fresh.